deploy/crds/kieapp.crd.yaml

```diff
@@ -83,12 +83,6 @@ spec:
                     amqClusterPassword:
                       type: string
                       description: The password to use for amq cluster user.
-                    controllerPassword:
-                      type: string
-                      description: The password to use for the controllerUser.
-                    serverPassword:
-                      type: string
-                      description: The password to use for the executionUser.
```

Comment:
I'm not sure if we can remove these items from the CRD until versions 7.5.x are gone from available/deployable versions in operator. Have you tested if those versions will deploy properly still?

Reply:
unless you can also change those version config files as well?

Reply:
Items have been removed from CRD but kept in the Go structs to prevent conflicts from 7.5.x.